communication/marketing-team/social-guidelines.md

```diff
@@ -0,0 +1,23 @@
+Future home of social media guidelines
+
+[insert twitter handle here]
+
+Examples:
+Try and tweet at least twice a week  
+No more than two original tweets a day, unlimited retweets; responding to others does not count in this  
+should respond within 48 hours
+Contributor information/news/stories is the target; forward others to social@cncf.io for @kubernetesio main twitter account  
+Bio should mention that this is powered by sig-contribex, link to the kubernetes/community repo  
+Never include someones photo, twitter handle, or other personal identifiable info without their explicit permission  
+try not to abbreviate as much as possible  
+all event tweets should have an action and/or thumbnail image    
```

Comment:
These are all con'cated together.  They need either an additional line or added as a list

Reply:
yep - just wanted folks to see the text first

Reply:
added a numbered list